openlibrary/catalog/get_ia.py

```diff
@@ -212,13 +219,14 @@ def get_from_archive(locator):
             if attempt == 4:
                 raise
             print 'retry, attempt', attempt
+    """
 
     r0, r1 = offset, offset+length-1
-    url = 'http://' + host + path + '/' + rest
+    url = 'https://archive.org/download/%s' % filename
```

Comment:
If possible, it would be great to have this as a config (as it was before)

Reply:
Can we extract archive.org into a variable and also not comment the code above but instead have find_item fall back to this if it fails:

```
def resolve_server(identifier):
    """
    Returns:
      a dict containing this item's:
        server: e.g. ia601507.us.archive.org
        dir: on disk containing files, e.g. /2/items/recurringwordsth00ethe
    """
    metadata = requests.get('%s/metadata/%s' % (API_BASEURL, identifier)).json()
    return {
        'dir': metadata['dir'],
        'server': metadata['server']
    }
``` 
https://github.com/internetarchive/openlibrary/issues/1060

Reply:
The case we care about is that we're not changing how this code works on production because whatever black magic this is works...

But because it breaks on dev, we want to at least provide a way to unblock developers in the case where the existing code path fails

Reply:
Let's bypass all of this by having an archive_org_url which is in the default openlibrary.yml config which we use here w/ the `/download` link